packages/spanner/test/session-pool.js

```diff
@@ -1533,55 +1533,55 @@ describe('SessionEvictor', function() {
 
   describe('evict', function() {
     var CONFIG = {};
-    var SESSION = {
-      evicted_: false,
-      run: util.noop
-    };
     var AVAILABLE = false;
+    var RESOURCE;
+
+    beforeEach(function() {
+      RESOURCE = {
+        obj: {
+          evicted_: false,
+          keepAlive: util.noop
+        }
+      };
+    });
 
     it('should return true if already evicted', function() {
-      var session = extend({}, SESSION, {
-        evicted_: true
-      });
+      RESOURCE.obj.evicted_ = true;
 
-      var evictReturnValue = sessionEvictor.evict(CONFIG, session, AVAILABLE);
+      var evictReturnValue = sessionEvictor.evict(CONFIG, RESOURCE, AVAILABLE);
       assert.strictEqual(evictReturnValue, true);
     });
 
     it('should evict from the DefaultEvictor', function(done) {
-      sessionEvictor.evictor.evict = function(config, session, available) {
+      sessionEvictor.evictor.evict = function(config, resource, available) {
         assert.strictEqual(config, CONFIG);
-        assert.strictEqual(session, SESSION);
+        assert.strictEqual(resource, RESOURCE);
         assert.strictEqual(available, AVAILABLE);
         done();
       };
 
-      sessionEvictor.evict(CONFIG, SESSION, AVAILABLE);
+      sessionEvictor.evict(CONFIG, RESOURCE, AVAILABLE);
     });
 
     it('should run a query & update evicted status on session', function(done) {
       sessionEvictor.evictor.evict = function() {
         return true;
       };
 
-      var session = extend({}, SESSION, {
-        run: function(query, callback) {
-          assert.strictEqual(query, 'SELECT 1');
-
-          callback(new Error('Error.'));
+      RESOURCE.obj.keepAlive = function(callback) {
+        callback(new Error('Error.'));
```

Comment:
I think we also need a test to see if the value is set properly when an error is not returned.